[2024-03-17 15:39:02,225] [INFO] [real_accelerator.py:191:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-03-17 15:39:04,320] [WARNING] [runner.py:202:fetch_hostfile] Unable to find hostfile, will proceed with training with local resources only.
[2024-03-17 15:39:04,321] [INFO] [runner.py:568:main] cmd = /cpfs01/user/chenqin.p/anaconda3/envs/moe/bin/python -u -m deepspeed.launcher.launch --world_info=eyJsb2NhbGhvc3QiOiBbMCwgMSwgMiwgM119 --master_addr=127.0.0.1 --master_port=53162 --enable_each_rank_log=None run_mlora.py --do_predict --data_config bigfive_task_test --peft_path /cpfs01/user/chenqin.p/dyh/MOELoRA-peft/saved/moelora/checkpoint-500/adapter_model.bin --test_file /cpfs01/user/chenqin.p/dyh/MOELoRA-peft/data/bigfive_questionnaire.xlsx --cache_dir data --overwrite_cache --prompt_column input --response_column target --model_name_or_path /cpfs01/user/chenqin.p/model_cached/Llama-2-7b-chat-hf --peft_path saved/moelora/checkpoint-4000 --output_dir results/pred/moelora --overwrite_output_dir --max_source_length 512 --max_target_length 256 --per_device_eval_batch_size 4 --predict_with_generate --lora_name moelora --expert_num 10 --task_num 32 --task_embedding_dim 64
Traceback (most recent call last):
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/runpy.py", line 187, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/runpy.py", line 110, in _get_module_details
    __import__(pkg_name)
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/deepspeed/__init__.py", line 10, in <module>
    import torch
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/__init__.py", line 1187, in <module>
    from .functional import *  # noqa: F403
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/functional.py", line 8, in <module>
    import torch.nn.functional as F
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/nn/__init__.py", line 1, in <module>
    from .modules import *  # noqa: F403
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/nn/modules/__init__.py", line 2, in <module>
    from .linear import Identity, Linear, Bilinear, LazyLinear
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/nn/modules/linear.py", line 7, in <module>
    from .. import functional as F
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/nn/functional.py", line 3676, in <module>
    def upsample(input: Tensor, size: Optional[List[int]] = None, scale_factor: Optional[float] = None, mode: str = "nearest", align_corners: Optional[bool] = None) -> Tensor:  # noqa: F811
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/_jit_internal.py", line 876, in _overload
    _check_overload_body(func)
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/_jit_internal.py", line 849, in _check_overload_body
    parsed_def = parse_def(func)
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/torch/_sources.py", line 126, in parse_def
    py_ast = ast.parse(dedent_src)
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
KeyboardInterrupt
Traceback (most recent call last):
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/bin/deepspeed", line 6, in <module>
    main()
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/site-packages/deepspeed/launcher/runner.py", line 584, in main
    result.wait()
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/subprocess.py", line 1209, in wait
    return self._wait(timeout=timeout)
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/subprocess.py", line 1959, in _wait
    (pid, sts) = self._try_wait(0)
  File "/cpfs01/user/chenqin.p/anaconda3/envs/moe/lib/python3.10/subprocess.py", line 1917, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt
